mne/io/eeglab/eeglab.py

```diff
@@ -258,6 +306,149 @@ def read_epochs_eeglab(input_fname, events=None, event_id=None, montage=None,
     return epochs
 
 
+def _bunchify(mapping, name='BU'):
+    """Convert mappings to Bunches recursively.
+
+    Based on https://gist.github.com/hangtwenty/5960435.
+    """
+    if isinstance(mapping, Mapping):
+        for key, value in list(mapping.items()):
+            mapping[key] = _bunchify(value)
+        return _bunch_wrapper(name, **mapping)
+    elif isinstance(mapping, list):
+        return [_bunchify(item) for item in mapping]
+    return mapping
+
+
+def _bunch_wrapper(name, **kwargs):
+    """Convert mappings to Bunches."""
+    return Bunch(**kwargs)
```

Comment:
`name` isn't even used, why have this function at all instead of directly constructing `Bunch` objects where it's used?